### ISSUE TITLE
Inconsistancy: DayOfWeek enum vs weekday int

### ISSUE THREAD

**Issue by frankhommers** [ocpi/ocpi#298]
I find different occurances of `DayOfWeek` vs `weekday`.
One is an enum, the other is an int. 

What is the reason?

**Comment by ahzf** [comment]
Where do so see "weekdays"?
If I remember correctly "day_of_week" was only used within charging tariff restrictions, or?

**Comment by frankhommers** [comment]
Yeah, I already fixed it by having two enums which are interchangeable in my code ;-)
But it's not nice to have different ways of representation.

**Comment by ahzf** [comment]
I had luck... in C# (and other languages) you can cast ENUMs to Integers ;)

But when you implement more of OCPI you will notice how much of OCPI is just roughly glued together without any ICT gardener harmonizing all those parts into a coherent architecture. All of those EV Roaming whatever and their "only we can gurantee a reliable protocol development" power point propaganda... and then all those flaws everywhere...  :-/

Anyway, when you are ready with some parts of your implementation, we could run our implementations against each other. I did this with a nice research institute here in Germany already and it was really a lot of fun... all those swearwords all the time ;)

**Comment by frankhommers** [comment]
Always good to get connected ;-) I'll add you on LinkedIn as well ;-)

**Comment by RobertdeLeeuw-IHomer** [comment]
@frankhommers you are the first to notice this. Has been part of OCPI since before 2.0. I guess it has not been used that much.
